application/migrations/m160714_114125_warning_set_unique_static_value_key.php

```diff
@@ -14,10 +14,9 @@ public function up()
     {
         $svTable = PropertyStaticValues::tableName();
         $slugsArray = (new Query())
-            ->select($svTable . '.*, count(id) as count')
+            ->select($svTable . '.slug, count(slug) as count')
             ->from($svTable)
             ->groupBy($svTable . '.slug')
-            ->having('count > 1')
```

Comment:
Referring [MySQL GROUP BY docs](https://dev.mysql.com/doc/refman/5.7/en/group-by-handling.html):
> Before MySQL 5.7.5, enabling ONLY_FULL_GROUP_BY disables this extension, thus requiring the HAVING clause to be written using unaliased expressions.

Therefore this trick should work and there will be no need in additional if on line 36